some.c

```diff
@@ -0,0 +1,3 @@
+system(string);
+system(string);
```

Comment:
:exclamation:**Cycode**: SAST violation: 'Locate all calls to interesting and potentially insecure API functions (candidate points). The auditor can backtrace from these candidate points to find pathways allowing access from untrusted input.'.
 
Severity: Medium


Would you like to exclude this SAST violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_sast_ignore_here** | Applies to this request only |
| **#cycode_sast_false_positive** | Applies to this SAST violation for all repos in your organization |

 <!-- 9ce7cf24-95bb-4269-bc27-bf7a4c4f250b -->